spikingjelly/activation_based/neuron/nonlinear_if.py

```
@@ -186,19 +186,21 @@ def multi_step_forward(self, x_seq: torch.Tensor):
+                    x_seq.flatten(1),
+                    self.v.flatten(0),
+                    self.tau,
+                    self.v_threshold,
+                    self.v_reset,
+                    self.v_rest,
+                    self.v_c,
+                    self.a0,
+                    self.detach_reset,
+                    self.surrogate_function,
+                )
+            except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)